lib/hmmlearn/base.py

```diff
@@ -253,6 +253,63 @@ def score(self, X, lengths=None):
         """
         return self._score(X, lengths, compute_posteriors=False)[0]
 
+    def score_sequences(self, X, lengths=None):
+        """
+        Compute the log probability under the model.
+        Compared to score() it returns a list of probabilities and doesn't add them up.
+
+        Parameters
+        ----------
+        X : array-like, shape (n_samples, n_features)
+            Feature matrix of individual samples.
+        lengths : array-like of integers, shape (n_sequences, ), optional
+            Lengths of the individual sequences in ``X``. The sum of
+            these should be ``n_samples``.
+
+        Returns
+        -------
+        logprobs : List[float]
+            Log likelihoods of sequences in ``X``.
+        """
+
+        _utils.check_is_fitted(self, "startprob_")
+        self._check()
+
+        X = check_array(X)
+        impl = {
+            "scaling": self._score_sequences_scaling,
+            "log": self._score_sequences_log,
+        }[self.implementation]
+        return impl(X=X, lengths=lengths)
+
+    def _score_sequences_log(self, X, lengths=None):
```

Comment:
I think you could modify _score_log/_score_scaling to not sum the individual scores (and then sum them in _score), and then you could reuse them here instead of duplicating the logic and having the IndexError specialcase just in score_sequences.

Reply:
Would you like all score methods to return -inf instead of IndexError?

Reply:
Returning -inf seems reasonable, as that's also what gets returned with `fit([[1]]); score([[0]])`.  It may be worth mentioning this behavior in the docstring of MultinomialHMM, too.